src/lib.rs

```diff
@@ -175,6 +175,10 @@ impl <K: KeyType, V: ValueType> BTree<K, V> {
         Ok( () )
     }
 
+
+    pub fn get(&self, key: &K) -> Option<&BTreeSet<V>> {
+        self.mem_tree.get(key)
+    }
```

Comment:
We could return `Option<Iter<&V>>` here instead and hide the `BTreeSet` from the caller. I think that might be better.